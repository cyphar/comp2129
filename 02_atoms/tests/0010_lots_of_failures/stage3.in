DISPLAY
DISPLAY 1 2 3
DISPLAY   
START 3 1 4
DISPLAY 3 2
DISPLAY    2
DISPLAY
DISPLAY 	
