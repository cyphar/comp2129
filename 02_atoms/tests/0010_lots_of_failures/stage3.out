Game Not In Progress

Game Not In Progress

Game Not In Progress

Game Ready
Red's Turn

Invalid Command

Invalid Command


+--+
|  |
|  |
|  |
|  |
+--+


+--+
|  |
|  |
|  |
|  |
+--+

